CHANGELOG.asciidoc

```diff
@@ -3,6 +3,54 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.17.4]]
+=== Beats version 8.17.4
+https://github.com/elastic/beats/compare/v8.17.3\...v8.17.4[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- The Beats logger and file output rotate files when necessary. The beat now forces a file rotation when unexpectedly writing to a file through a symbolic link.
+- Allow faccessat(2) in seccomp. {pull}43322[43322]
+
+*Filebeat*
+
+- Fixed race conditions in the global ratelimit processor that could drop events or apply rate limiting incorrectly.
```

Comment:
This isn't a breaking change it's a bug fix. The linked PR is https://github.com/elastic/beats/pull/42966